### ISSUE TITLE
You have to work on this!!!

### ISSUE THREAD

**Issue by ghost** [meteorcrowd/crowducate-next#152]
Update your current one. This one is sooooooooo amazing and sooooooo much better.............. also I would suggest changing ure css for the home page for crowducate.me

**Comment by Rahbaran** [comment]
You are right. We're working on it. The development branch of the [main repo](https://github.com/Crowducate/crowducate-platform/tree/develop) is the continuation. Soon, we'll delete this repo. Sorry for the inconvenience.

**Comment by brylie** [comment]
@android-fanatic thanks for the positive feedback :-)
